spoom.gemspec

```
@@ -37,6 +37,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency("rbi", ">= 0.3.1")
   spec.add_dependency("sorbet-static-and-runtime", ">= 0.5.10187")
   spec.add_dependency("thor", ">= 0.19.2")
+  spec.add_dependency("rexml")
```

Let's put a lower bound, possibly 3.2.6?

```suggestion
  spec.add_dependency("rexml", ">= 3.2.6")
```